### ISSUE TITLE
Ligatures don't work with powerlevel10k

### ISSUE THREAD

**Issue by Igorgro** [tonsky/FiraCode#1053]
Powerline symbols, as example branch icon, dont work in powerlevel10k. This is what shown in my console:
![изображение](https://user-images.githubusercontent.com/22894169/82710215-09fa3080-9c8b-11ea-8571-efef12ed4c6b.png)
This is not a problem of powerlevel10k, because their own MesloLGS NF font works well.

**Comment by tonsky** [comment]
Yes. Powerline uses U+E0A0, but Powerlevel10k set up U+F126 instead. Both are in private use area, so there’s no standard on those, but U+E0A0 seems to be more popular? In fact, MesloLGS NF seems to be the only font providing U+F126.

<img width="800" alt="Screenshot 2020-05-22 at 23 46 20" src="https://user-images.githubusercontent.com/285292/82711859-e2a16480-9c86-11ea-8f86-07dac2b23424.png">


If you want to fix it for yourself:

```
  typeset -g POWERLEVEL9K_VCS_BRANCH_ICON='\uE0A0 '
```

If you want to contribute to the community, open an issue with Powerlevel10k to use same branch icon that every other programmer font already provides.

**Comment by hensz** [comment]
Hi!
Just curious, if the icons are present but with a different code, does this mean I can use powerlevel10k fully with Fira Code by simply redefining some variables? If yes, is there a list of variable and value? This would greatly help people like me that are not that firm in unicode stuff ;-)

**Comment by romkatv** [comment]
> Yes. Powerline uses U+E0A0, but Powerlevel10k set up U+F126 instead. Both are in private use area, so there’s no standard on those, but U+E0A0 seems to be more popular? In fact, MesloLGS NF seems to be the only font providing U+F126.

This isn't quite right.

- U+E0A0 is "branch" in Powerline.
- U+F126 is "fork" in Font Awesome.
- All Nerd Fonts have both of these.
- MesloLGS NF is a Nerd Font.
- When powerlevel10k runs for the first time or when you manually run `p10k configure` it displays certain symbols and asks the user how they look. One of these questions ascertains whether your font has Font Awesome glyphs. Only if you confirm this, powerlevel10k may use U+F126.

tl;dr: FiraCode doesn't have Font Awesome glyphs. Run `p10k configure` and answer questions correctly to let Powerlevel10k know which glyphs your font has. Powerlevel10k won't use glyphs that you font doesn't have.
